indexer/services/ender/src/lib/helper.ts

```diff
@@ -91,6 +91,7 @@ export function dateToDateTime(
  * @returns
  */
 export function indexerTendermintEventToEventProtoWithType(
+  blockEventIndex: number,
```

Comment:
Each event in the block is stored on the handler so that the corresponding row index is passed to said handler.

Is there a better way I could do this mapping?

Reply:
Makes sense, can you add a comment describing this property?